src/app/lib/config/services/serbian.js

```diff
@@ -5,6 +5,8 @@ import {
 } from '@bbc/gel-foundations/scripts';
 import { serbian as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/GMT';
+import '@bbc/psammead-locales/moment/sr-cyrl';
+import '@bbc/psammead-locales/moment/sr';
```

Comment:
The locales used (at the bottom of this file) are `sr-latn` (not `sr`) and `sr-cyrl`

Reply:
Is this the right thing to have done here? Use `datetimeLocale` of `sr` (latin characters) for the `lat` service and `sr-cyrl` for `cyr` service.